website/docs/r/volume.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "cloudscale"
+page_title: "cloudscale.ch: cloudscale_volume"
+sidebar_current: "docs-cloudscale-resource-volume"
+description: |-
+  Provides a cloudscale.ch Volume (block storage) resource. This can be used to create, modify, and delete Volumes.
+---
+
+# cloudscale\_volume
+
+Provides a cloudscale.ch Volume (block storage) resource. This can be used to create, modify, and delete Volumes.
+
+## Example Usage
+
+```hcl
+# Create a new Server
+resource "cloudscale_server" "web-worker01" {
+  name        = "web-worker01"
+  flavor_slug = "flex-4"
+  image_slug  = "debian-9"
+  ssh_keys    = ["ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIL2jzgla23DfRVLQr3KT20QQYovqCCN3clHrjm2ZuQFW user@example.com"]
+}
+
+# Add a Volume to web-worker01
+resource "cloudscale_volume" "web-worker01-vip" {
```

Comment:
Can you rename the resource to e.g. `web-worker01-volume` like in the `index.html.markdown`? The current name seems to be a copy/paste from the floating IP documentation.